catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/ChartRepository.java

```diff
@@ -59,6 +59,7 @@ public static String getFQN(Chart chart) {
 
   @Override
   public void prepare(Chart chart) throws IOException, ParseException {
+    EntityUtil.escapeReservedChars(getEntityInterface(chart));
```

Comment:
Instead of passing Object and List of Objects. Then doing type casting why not just send the strings.
Since you are calling this method explicitly anyway in the each of the Repository. Might as well call all the names example table and table columns can looped and call with escapeReservedChars

Reply:
Lets merge this in, we can refactor later.

Reply:
Ok, let's refactor later.